interfaces/rerank/README.md

```diff
@@ -0,0 +1,64 @@
+`@supplies` directive connects a field selection on a concrete type to an interface request. 
```

Comment:
No it doesn't :-)

Reply:
ha -- I thought I copied that from the documentation!